tests/bugs/generic-type-arg-overloaded.slang.expected

```diff
@@ -1,11 +1,17 @@
 result code = -1
 standard error = {
 tests/bugs/generic-type-arg-overloaded.slang(14): error 30200: declaration of 'Stuff' conflicts with existing declaration
+struct Stuff {}
+^~~~~~
```

Comment:
This is a case where it would be great to have the diagnostic indicate the `Stuff` identifier. This comes down to our decision right now to have the location of a declaration usually point at the declaration keyword (`struct`, `var`, etc.) instead of the name (it stores a separate `NameLoc` that can identify the location of the name token).